src/Sylius/Bundle/AdminBundle/Resources/config/app/sylius/sylius_mailer.yml

```diff
@@ -9,3 +9,6 @@ sylius_mailer:
         shipment_confirmation:
             subject: sylius.emails.shipment_confirmation.subject
             template: "@SyliusAdmin/Email/shipmentConfirmation.html.twig"
+        admin_password_reset:
```

Comment:
Alphabetical order R.I.P [*]

Reply:
Even tho, we are discussing "admin" in this PR, we are working on API so these translations should go either to Core or Api